govcd/catalog_test.go

```diff
@@ -435,7 +444,7 @@ func (vcd *TestVCD) Test_CatalogUploadMediaImage_ShowUploadProgress_works(check
 
 	AddToCleanupList(itemName, "mediaCatalogImage", vcd.org.Org.Name+"|"+vcd.config.VCD.Catalog.Name, "Test_CatalogUploadMediaImage_ShowUploadProgress_works")
 
-	check.Assert(string(result), Matches, ".*Upload progress 100.00%")
+	check.Assert(string(result), Matches, ".*Upload progress 100.00%.*")
```

Comment:
I get errors in these two tests - 

```
START: catalog_test.go:419: TestVCD.Test_CatalogUploadMediaImage_ShowUploadProgress_works
Running: TestVCD.Test_CatalogUploadMediaImage_ShowUploadProgress_works
catalog_test.go:447:
    check.Assert(string(result), Matches, ".*Upload progress 100.00%.*")
... value string = "Waiting...\rUpload progress 0.00%\rUpload progress 100.00%\n"
... regex string = ".*Upload progress 100.00%.*"

FAIL: catalog_test.go:419: TestVCD.Test_CatalogUploadMediaImage_ShowUploadProgress_works
```


```
START: catalog_test.go:202: TestVCD.Test_UploadOvf_ShowUploadProgress_works
Running: TestVCD.Test_UploadOvf_ShowUploadProgress_works
catalog_test.go:230:
    check.Assert(string(result), Matches, ".*Upload progress 100.00%")
... value string = "Waiting...\rUpload progress 0.00%\rUpload progress 100.00%\n"
... regex string = ".*Upload progress 100.00%"

FAIL: catalog_test.go:202: TestVCD.Test_UploadOvf_ShowUploadProgress_works
```

Reply:
Sorry about that. My vCloud ENV has some issue so I didn't run test completely. Per [the article here](https://github.com/google/re2/wiki/Syntax), dot in regex doesn't mathc new line \n by default. So, I enabled new line matching with s flag in the new code.

Reply:
Np - thanks. I will try to run some tests and see if it doesn't break any existing stuff.